backend/fms_core/filters.py

```diff
@@ -189,6 +190,12 @@ def latest_release_update_filter(self, queryset, name, value):
             latest_release_update=Max("readsets__release_status_timestamp")
         ).filter(latest_release_update__gt=value)
 
+    def latest_validation_status_update_filter(self, queryset, name, value):
+        return queryset.annotate(
+            latest_validation_status_update=Max("readsets__validation_status_timestamp")
+        ).filter(latest_validation_status_update__gt=value)
+
+
```

Comment:
Wasn't this code from sebastian's PR?

Reply:
Yes. Yes it is.

Reply:
yeah i spotted some cross polination in your PR sebastian be careful where you spread it ... you can play in my PR but make sure not to add unrelated stuff of your own (i keep that right to myself ;).

Reply:
the other pr was really just the endpoint and in order to test ulysses changes out, i decided to move the code here, 
the options proposed:
1. leave here and delete my pr (since this is the only change that it has)
2. i can remove this one line and we push the other pr and this one separately

any option goes for me. i would always go for simplicity but i leave to the commenters to decide

Reply:
i closed the other pr and kept the validation related code in this PR